### ISSUE TITLE
Long Updating...

### ISSUE THREAD

**Issue by bolfox** [homebridge/homebridge#1832]
I have an homekit instance on a rpi3, plus knx interface.

Sometimes when i open Home app on ios, it keep Updating... status for about 60 second, after 60s tell that no one accessory is responding, and after one second everithing works...

Ideas?

Tnx

**Comment by beniaminrychter** [comment]
I have exact same trouble. It's so frustrating. @bolfox is everything works normally on LTE or just outside your home WiFi network?

**Comment by bolfox** [comment]
I use only in wifi...

**Comment by luvmex** [comment]
Same as mine situation, It has to wait for few mins to get everything work but sometimes get no respond and that will need to reload again. This will happens on both home wifi and remote network.

@bolfox  What other plugins(accessories) did you install?

**Comment by beniaminrychter** [comment]
@luvmex Oh, that's weird that it's not working on remote network. Are you using Apple TV as your bridge? 

My problem is gone. I had to change my WiFi router and now everything is fine, super quick response from Homebridge. Issue is connected with Multicasting [`](https://wiki.openwrt.org/doc/howto/udp_multicast). I used some cheap TPLink with OpenWrt custom firmware. Now I bought also cheap D-Link but with official firmware and is working like a gold :)
